### PR TITLE
Add annotate to project

### PR DESCRIPTION
**Pull request by flomosq**
# Trello Card #35

Add annotate and auto annotate models

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -34,6 +34,7 @@ group :development do
   gem 'listen', '~> 3.3'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
+  gem 'annotate'
```
Comment (martinjaimem) [inline Gemfile]: You can add the version you installed, like the one 3 lines up: `gem 'listen', '~> 3.3'`

Comment (martinjaimem) [reply Gemfile]: In this case it would be `3.1.1` (you can check your gemfile.lock)

**Review by JuanfraM — APPROVED** [review]

File: Gemfile (hunk 1 above)
Comment (flomosq) [reply Gemfile]: Nice, and why doesn't spring have the version? Is that intended?

Comment (martinjaimem) [reply Gemfile]: That's a really good question 😆 . I don't know, in the meantime, feel free to merge this PR without changing `gem 'spring'`

Comment (flomosq) [reply Gemfile]: ok great 😄

**Review by santir489 — APPROVED** [review]
Nice!

**Review by martinjaimem — APPROVED** [review]

**Pull request merged**